modules/API/external/external_api.js

```diff
@@ -1,80 +1,66 @@
-const logger = require("jitsi-meet-logger").getLogger(__filename);
+import EventEmitter from 'events';
+import postisInit from 'postis';
 
-/**
- * Implements API class that embeds Jitsi Meet in external applications.
- */
-
-var postisInit = require("postis");
+const logger = require('jitsi-meet-logger').getLogger(__filename);
 
 /**
- * The minimum width for the Jitsi Meet frame
- * @type {number}
+ * Maps the names of the commands expected by the API with the name of the
+ * commands expected by jitsi-meet
  */
-var MIN_WIDTH = 790;
+const commands = {
+    'avatarUrl': 'avatar-url',
+    'displayName': 'display-name',
+    'email': 'email',
+    'hangup': 'video-hangup',
+    'toggleAudio': 'toggle-audio',
+    'toggleChat': 'toggle-chat',
+    'toggleContactList': 'toggle-contact-list',
+    'toggleFilmStrip': 'toggle-film-strip',
+    'toggleShareScreen': 'toggle-share-screen',
+    'toggleVideo': 'toggle-video'
+};
 
 /**
- * The minimum height for the Jitsi Meet frame
- * @type {number}
+ * Maps the names of the events expected by the API with the name of the
+ * events expected by jitsi-meet
  */
-var MIN_HEIGHT = 300;
+const events = {
+    'displayNameChange': 'display-name-change',
+    'incomingMessage': 'incoming-message',
+    'outgoingMessage': 'outgoing-message',
+    'participantJoined': 'participant-joined',
+    'participantLeft': 'participant-left',
+    'readyToClose': 'video-ready-to-close',
+    'videoConferenceJoined': 'video-conference-joined',
+    'videoConferenceLeft': 'video-conference-left'
+};
```

Comment:
While your gonna be all over the blame anyway, maybe it's time to get rid of the quotations around the keys (if that's the preferred style).